main.js

```diff
@@ -0,0 +1,37 @@
+
+var suggestedFood = document.querySelector('.you-should-make');
+var letsCookButton = document.querySelector(".lets-cook-button");
+var cookpot = document.querySelector(".cookpot");
+var rightBox = document.querySelector(".right-box");
+
+
+letsCookButton.addEventListener('click', checkingRadioButtons)
+
+function checkingRadioButtons() {
+  event.preventDefault();
+  var checkedDish;
+  var foods = document.querySelectorAll("input[name='food']");
+  for(var i = 0; i < foods.length; i++){
+    if(foods[i].checked){
+      checkedDish = foods[i].value;
+      break;
+    };
+  };
+  if(checkedDish === "side"){
```

Comment:
consider splitting this function up? ex: checkRadioButtons() / displayText() 
Usual naming conventions on functions are named after what it is the function is executing....

Reply:
Thanks Kevin. This is exactly what I have been trying to figure out how to do is split into 2 functions. Haven't quite gotten there yet...